pkgs/games/kabeljau/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, lib, fetchFromGitea, bash, dialog, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "kabeljau";
+  version = "unstable-2022-04-16";
```

Comment:
If this is your project cant you just make an 0.0.1 release?

Id also say since this is quite niche this should go in a NUR https://github.com/nix-community/NUR
Or add a default.nix/flake.nix to the repo to package it

Reply:
> d also say since this is quite niche this should go in a NUR https://github.com/nix-community/NUR
Or add a default.nix/flake.nix to the repo to package it

Why can't it go in Nixpkgs? There are many niche package in Nixpkgs. Making a flake in the project's repo just makes it less convenient to install.

Reply:
Because there are only so many maintainers and if you update this someone has to review it.

There are minor increases in cost like evaluation time increase, git repo size increase, more things to build on Hydra, more things to cache in cache.nixos.org
This may be minor for each program (especially something more light weight like a script) but if there are hundreds and hundreds it adds up.

In no way do I wish to diminish or disparage your project, it looks fun, but not everything is suited to nixpkgs, thats why idea of the NUR exists
The NUR or a flake is still a very easy way to install a package.